improver_tests/utilities/test_spatial.py

```diff
@@ -433,7 +435,11 @@ class Test_transform_grid_to_lat_lon(IrisTest):
 
     def setUp(self):
         """Set up the cube."""
-        self.cube = set_up_cube(zero_point_indices=((0, 0, 1, 1),), num_grid_points=2)
+        data = np.ones((1, 2, 2), dtype=np.float32)
+        data[:, 1, 1] = 0.0
+        self.cube = set_up_variable_cube(
+            data, "precipitation_amount", "kg m^-2", "equalarea",
+        )
         self.cube.coord(axis="x").points = np.array([-1158000.0, 924000.0])
         self.cube.coord(axis="y").points = np.array([-1036000.0, 902000.0])
 
```

Comment:
Both unused data and need to use `grid_spacing` and `domain_corner` arguments here.

Reply:
To use the set_up_variable_cube method I have had to modify the test data. The grid was not previously square, and the setup function only allows for square grids.